modules/serverpod_auth/serverpod_auth_google_flutter/lib/src/google/sign_in_with_google_mobile.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/foundation.dart';
+import 'package:google_sign_in/google_sign_in.dart';
+import 'package:serverpod_auth_google_flutter/src/google/sign_in_with_google_platform.dart';
+
+/// Atempts to signin with google on mobile platforms and returns the auth tokens.
```

Comment:
I think we should point out that this method is typically internal to Serverpod so users don't attempt to call it directly. Maybe we should see if it's worth adding an`@internal` annotation here to.

Reply:
(Same in the web file.)